lib/adapters/parsers/csv.js

```diff
@@ -33,7 +33,7 @@ class CSVParser extends base {
                     self.doneParsing = true;
                     emit(buffer);
                     buffer = [];
-                    csvStream.end();
+                    csvStream.emit('end');
```

Comment:
I think this was from fiddling to understand why that other tests was failing due to the "internal buffer size changes"